packages/esm-patient-chart-app/src/visit/visit-form/visit-attribute-type.component.tsx

```diff
@@ -131,13 +120,12 @@ const AttributeTypeField: React.FC<AttributeTypeFieldProps> = ({
 
         return (
           <Select
+            {...fieldProps}
             labelText={labelText}
-            onChange={(e) => setVisitAttribute(e.target.value)}
-            required={required}
-            invalid={required && isMissingRequiredAttributes && !visitAttributes[uuid]}
+            invalid={!!errors.visitAttributes?.[uuid]}
             invalidText={t('fieldRequired', 'This field is required')}
           >
-            <SelectItem text={t('selectAnOption', 'Select an option')} value={null} />
+            <SelectItem text={t('selectAnOption', 'Select an option')} value={''} />
```

Comment:
Is this necessary? Seems better that the `selectAnOption` option isn't usable.

Reply:
Actually, since the value is null, when "Select an option" is selected, instead of setting the value in the state as null, it saves "Select an option", which is truthy.

The user can select it again, if that is what you are afraid of

Reply:
P.S. This is the Ozone distro cambodia environment, used for testing purposes.
https://github.com/openmrs/openmrs-esm-patient-chart/assets/51502471/9b41503c-b8f9-4833-91cd-384743ff45c7